kong/cmd/utils/nginx_signals.lua

```diff
@@ -8,11 +8,22 @@ local pl_stringx = require "pl.stringx"
 local fmt = string.format
 
 local nginx_bin_name = "nginx"
-local nginx_search_paths = {
-  "/usr/local/openresty/nginx/sbin",
-  "/opt/openresty/nginx/sbin",
-  ""
-}
+local nginx_search_paths
+do
+  local resty_prefix = os.getenv("KONG_OPENRESTY_PREFIX")
```

Comment:
Should this be made a kong config property instead? It would fit nicely the miscellaneous category that includes `lua_package_path`. Property could be named `openresty_prefix`; this way, it would be specified as either the property or `KONG_OPENRESTY_PREFIX` env var.